src/twp-editor-plugin/files/plugin-state-templates/pm-plugins/plugin-factory.ts

```diff
@@ -7,4 +7,4 @@ export const {
   createCommand,
   getPluginState,
   createPluginState,
-} = pluginFactory(<%= camelize(name) %>PluginKey, reducer);
+} = pluginFactory(<%= camelize(name) %>PluginKey, reducer /**, { mapping, onDocChanged, onSelectionChanged } **/);
```

Comment:
_Tiny_ change, but I think it'd be better if the trailing comma was included, as Prettier would add it anyway inside the AF repo